sdk/modelsrepository/azure-iot-modelsrepository/src/main/java/com/azure/iot/modelsrepository/ModelsRepositoryAsyncClient.java

```diff
@@ -62,4 +63,64 @@
     public ModelsRepositoryServiceVersion getServiceVersion() {
         return this.serviceVersion;
     }
+
+    /**
+     * Gets a collection of model definitions.
+     *
+     * @param dtmi A well-formed DTDL model Id. See <a href="https://github.com/Azure/opendigitaltwins-dtdl/blob/master/DTDL/v2/dtdlv2.md">DTDL specs</a>. For example 'dtmi:com:example:Thermostat;1'.
+     * @return A Map containing the model definition(s) where the key is the dtmi
+     * and the value is the raw model definition string.
+     */
+    @ServiceMethod(returns = ReturnType.SINGLE)
+    public Mono<Map<String, String>> getModels(String dtmi) {
+        return getModels(dtmi, this.defaultDependencyResolutionOption);
+    }
+
+    /**
+     * Gets a collection of model definitions.
+     *
+     * @param dtmi                 A well-formed DTDL model Id. See <a href="https://github.com/Azure/opendigitaltwins-dtdl/blob/master/DTDL/v2/dtdlv2.md">DTDL specs</a>. For example 'dtmi:com:example:Thermostat;1'.
+     * @param dependencyResolution A {@link ModelsDependencyResolution} value to force model resolution behavior.
```

Comment:
Is "force" right word or "define"?

Reply:
I agree, I will use dictate ... how does that sound?